docs/source/contributing.rst

```diff
@@ -150,7 +150,7 @@ When the upstream `mapd-core`_ project updates its Apache Thrift definition file
    cd ./heavydb
 
    # Use Thrift to generate the Python bindings
-   thrift -gen py -r omnisci.thrift
+   thrift -gen py -r heavydb.thrift
```

Comment:
Referring to https://github.com/heavyai/heavydb-internal/blob/master/heavy.thrift
```suggestion
   thrift -gen py -r heavy.thrift
```

Reply:
Ark yes should have updated the name of this file maybe. Not a big deal.